kivy/uix/widget.py

```diff
@@ -696,7 +696,7 @@ def clear_widgets(self, children=None):
         for child in children:
             remove_widget(child)
 
-    def export_to_png(self, filename, *args, **kwargs):
+    def export_to_png(self, filename, format='png', *args, **kwargs):
```

Comment:
we usually use fmt to avoid shadowing `format` which is a builtin function (not a big deal, in itself, but it's better to be consistent).

Reply:
That's fair, was avoiding fmt=fmt.